recipes/px4tools/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "px4tools" %}
+{% set version = "0.4.5" %}
```

Comment:
I'm really confused by this. Tags v0.3.7...v0.4.5 are all released on the same day. Also only v0.3.5 is on PyPI, but not of these later tags. Could we please ping the author and ask him to put sdists for these other tags on PyPI?